evaluate_library_access_from_output_tsv/manual-doi-checks.tsv

```diff
@@ -1,201 +1,201 @@
+"10.1080/00377996.1953.9957299"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1038/s41550-016-0030"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1111/j.1478-4408.1958.tb02258.x"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.1111/j.1550-7408.1962.tb02648.x"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.1007/978-3-658-12388-8_1"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1017/s1357729800051109"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1108/s1479-3679(2013)0000020014"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1136/bmj.1.4706.586"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.1093/molehr/gaq017"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1093/mnras/89.4.329"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1093/qjmed/hct203"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1097/acm.0000000000000545"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.3109/17453059209051382"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.2165/00003495-198300252-00083"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1007/978-1-4614-2251-8_15"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1179/136404609x12572514498892"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.3855/jidc.4620"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1017/s0007485307005305"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1016/0021-9517(79)90166-0"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1002/14651858.cd008009.pub2"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.1007/978-94-011-4683-8_4"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1002/prac.18430290165"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.1016/0306-2619(90)90086-s"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1680/geot.2004.54.3.233"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1111/j.1468-5914.1986.tb00063.x"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.3934/dcdsb.2014.19.485"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1097/ccm.0b013e31821b85c6"	0	2018-01-04	"1"	2018-01-04	"0"
+"10.1515/ci.2008.30.1.8"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1080/00222338008068111"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1080/00150190902848156"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1007/bf01557174"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1080/14786419.2014.988714"	0	2018-01-04	"0"	2018-01-04	"0"
+"10.1016/j.scient.2011.05.025"	0	2018-01-04	"1"	2018-01-04	"1"
+"10.1051/eas/1042000"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.3727/096368912x655154"	0	2018-01-03	"1"	2018-01-04	"1"
+"10.1007/bf03164593"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1109/ijcnn.2007.4371306"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1007/s00261-016-0956-8"	0	2018-01-03	"1"	2018-01-04	"0"
+"10.1017/s0020860400011542"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1587/transele.e92.c.1504"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1007/bf01678469"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1107/s0108767388009286"	0	2018-01-03	"1"	2018-01-04	"0"
+"10.1080/0889311x.2014.973868"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1016/s0171-2985(80)80037-4"	0	2018-01-03	"0"	2018-01-04	"0"
+"10.1002/chin.197531174"	0	2018-01-03	"1"	2018-01-04	"0"
+"10.1002/zaac.19402430401"	0	2018-01-03	"1"	2018-01-04	"0"
+"10.1029/2011jd016541"	0	2018-01-03	"1"	2018-01-04	"1"
+"10.17816/jowd6265-11"	0	2018-01-03	"invalid"	2018-01-04	"1"
```

Comment:
See https://github.com/greenelab/library-access/pull/18#discussion_r159524397. The website was temporarily down. Let's redo the on-campus check now that the site is back up.

Reply:
Sure. Done in 78d1654174.